### PR TITLE
Tabs: Fix style issues

### PR DESCRIPTION
**Pull request by jzaefferer**
Powered by esformatter! Will do more of these with individual files.

### CODE REVIEW

File: ui/tabs.js [hunk 1]
```diff
@@ -410,27 +410,27 @@ return $.widget( "ui.tabs", {
...
+			} );
 		this._addClass( this.tabs, "ui-tab", "ui-state-default" );
 
-		this.anchors = this.tabs.map(function() {
-				return $( "a", this )[ 0 ];
-			})
-			.attr({
+		this.anchors = this.tabs.map( function() {
+			return $( "a", this )[ 0 ];
+		} )
```
Comment (scottgonzalez) [inline ui/tabs.js]: I think we either need to clarify this in our style guide or change how this is formatted. The current rule is:

> When a chain of method calls is too long to fit on one line, there must be one call per line, with the first call on a separate line from the object the methods are called on. If the method changes the context, an extra level of indentation must be used.

So, presumably this should be:

```js
this.anchors = this.tabs
    .map( function() {
        return $( "a", this )[ 0 ];
    } )
        .attr( {
            ...
        } );
```

Comment (scottgonzalez) [inline ui/tabs.js]: There are other instances of this below.

**Comment by jzaefferer** [comment]
Updated with a second commit. Addresses several cases where line breaks were missing. If these changes are correct, I can look into update my esformatter-jquery-chain plugin to address those line breaks as well.

**Comment by scottgonzalez** [comment]
I don't remember where we had the conversation, but I thought we decided that filtering as the first action was allowed on the first line.

/cc @mikesherov

**Comment by jzaefferer** [comment]
That's pretty random - both `find` and `filter` change context.

Can I merge the first commit and open a new PR for the other?

**Comment by scottgonzalez** [comment]
> That's pretty random - both find and filter change context.

And `.not()` and `.children()` and lots of other methods. I thought this existing block was valid:

```js
this.tabs.not( this.active ).attr( {
	"aria-selected": "false",
	"aria-expanded": "false",
	tabIndex: -1
} );
```

Or maybe:

```js
this.tabs.not( this.active )
	.attr( {
		"aria-selected": "false",
		"aria-expanded": "false",
		tabIndex: -1
	} );
```

But I didn't expect it to be:

```js
this.tabs
	.not( this.active )
		.attr( {
			"aria-selected": "false",
			"aria-expanded": "false",
			tabIndex: -1
		} );
```

Though I understand that's the rule as written.

**Comment by dmethvin** [comment]
The first looks best to me as a reader and is still very clear. I'm not sure how the style guide should balance human readability vs absolute rules that can be enforced by a formatter, but it seems sad to lose the former in search of the latter.

**Comment by jzaefferer** [comment]
My second commit was created by hand, no esformatter involved. Both esformatter and my esformatter-jquery-chain plugin tolerate line breaks as-is. The plugin only fixes indent.

**Comment by jzaefferer** [comment]
I merged my first commit, created a new PR for the other, #1508

**Pull request merged**